tests/unit/test_ods.py

```diff
@@ -0,0 +1,214 @@
+import mock
+import unittest
+import functools
+
+import app.helpers.ods as ods
+
+from hypothesis import strategies as st
+from hypothesis import given, example, assume
+
+po = functools.partial(mock.patch.object, autospec=True)
```

Comment:
its short because patch.object along with the keys ends up being stupid long, and i didn't want to add a global randomly to the other test file.